vignettes/forecast_data.Rmd

```diff
@@ -0,0 +1,265 @@
+---
+title: "Example forecast hub data"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Example forecast hub data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.height = 5,
+  fig.width = 8,
+  fig.align = "center"
+)
+```
+
+```{r setup}
+library(hubExamples)
+library(hubVis)
+library(dplyr)
+library(ggplot2)
+```
+
+# Overview
+
+The `hubExamples` package provides three data sets that contain example model output and
+target data for an example forecast hub: `forecast_outputs`, `forecast_target_ts`, and
+`forecast_target_observations`. These forecasts and target data are a subset of the model outputs and target data that are provided in the [example-complex-forecast-hub](https://github.com/Infectious-Disease-Modeling-Hubs/example-complex-forecast-hub). These data were in turn derived from forecast submissions and target data for the [FluSight Forecast Hub](https://github.com/cdcepi/Flusight-forecast-data) for the 2022/23 season.
+
+We begin with a high level overview of these data objects and then we describe the different forecast targets in more detail.
+
+## Example forecast output data
+
+The example forecasts provided in `forecast_outputs` are derived from forecasts that were submitted to the FluSight hub from three models: `Flusight-baseline`, `MOBS-GLEAM_FLUH`, and `PSI-DICE`. The original forecasts submitted to the hub were in quantile format, but we have modified those submissions to provide examples of additional model output types and targets. The predictions for these other output types should be viewed only as illustrations of the data formats, not as real examples of forecasts. We will describe the methods used for creating other forecast output types below.
+
+The snippet below shows the format of the `forecast_outputs`.
+
+```{r}
+head(forecast_outputs)
+str(forecast_outputs)
+```
+
+This is a data frame with four groups of columns (see the [hubverse documentation](https://hubverse.io/en/latest/user-guide/model-output.html) for more information about these data formats):
+
+1. The `model_id` identifies the model that produced the predictions.
+2. Together, the `location`, `reference_date`, `horizon`, `target_end_date`, and `target` columns serve to identify a prediction task:
```

Comment:
Noting that these columns can be called the task IDs may be helpful for consistency across hubverse materials